config.json

```diff
@@ -288,296 +307,395 @@
-        "uuid": "bf30b17f-6b71-4bb5-815a-88f8181b89ae",
-        "practices": [],
+        "slug": "rotational-cipher",
+        "name": "Rotational Cipher",
+        "uuid": "4c408aab-80b9-475d-9c06-b01cd0fcd08f",
+        "practices": ["strings"],
+        "prerequisites": ["basics", "conditionals", "numbers", "strings"],
+        "difficulty": 1
+      },
+      {
+        "slug": "binary-search",
+        "name": "Binary Search",
+        "uuid": "a8288e93-93c5-4e0f-896c-2a376f6f6e5e",
+        "practices": ["loops"],
         "prerequisites": [
           "basics",
           "bools",
           "conditionals",
           "lists",
+          "list-methods",
           "loops",
-          "sets",
           "strings",
           "string-methods"
         ],
+        "difficulty": 1
+      },
+      {
+        "slug": "list-ops",
+        "name": "List Ops",
+        "uuid": "818c6472-b734-4ff4-8016-ce540141faec",
+        "practices": ["list-methods"],
+        "prerequisites": ["conditionals", "lists", "list-methods", "loops"],
+        "difficulty": 1
+      },
+      {
+        "slug": "acronym",
+        "name": "Acronym",
+        "uuid": "038c7f7f-02f6-496f-9e16-9372621cc4cd",
+        "practices": ["string-methods"],
+        "prerequisites": ["basics", "loops", "strings", "string-methods"],
+        "difficulty": 2
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "cb581e2c-66ab-4221-9884-44bacb7c4ebe",
+        "practices": ["comparisons"],
+        "prerequisites": ["basics", "numbers", "comparisons", "conditionals"],
+        "difficulty": 2
+      },
+      {
+        "slug": "pig-latin",
+        "name": "Pig Latin",
+        "uuid": "a5bc16da-8d55-4840-9523-686aebbaaa7e",
+        "practices": ["conditionals"],
+        "prerequisites": ["basics", "bools", "conditionals"],
         "difficulty": 2
       },
       {
         "slug": "protein-translation",
         "name": "Protein Translation",
         "uuid": "c89243f3-703e-4fe0-8e43-f200eedf2825",
-        "practices": ["dicts", "dict-methods", "loops"],
+        "practices": ["loops"],
         "prerequisites": [
           "basics",
           "conditionals",
-          "dicts",
           "lists",
           "loops",
           "strings",
           "string-methods"
         ],
-        "difficulty": 3
+        "difficulty": 2
       },
       {
-        "slug": "armstrong-numbers",
-        "name": "Armstrong Numbers",
-        "uuid": "d9ceb246-b518-42b9-9fa3-112e25c7ecd8",
-        "practices": ["generator-expressions"],
+        "slug": "scrabble-score",
+        "name": "Scrabble Score",
+        "uuid": "d081446b-f26b-41a2-ab7f-dd7f6736ecfe",
+        "practices": ["dicts"],
         "prerequisites": [
           "basics",
           "lists",
           "loops",
-          "numbers",
+          "dicts",
           "strings",
           "string-methods"
         ],
-        "difficulty": 1
+        "difficulty": 2
```

Comment:
Both you and Isaac rated `Scrabble Score` a 2 on the [google sheet](https://docs.google.com/spreadsheets/d/1ZrPNvsD2buxKVamU86k9taYNqtz4TEi6cd3ptkxST6Q/edit#gid=0).  It was a `3`.  The removed section on the left is for `armstrong-numbers`.  😄

Reply:
Yeah, that's probably one that has seemed simpler the more languages I've solved it in. Which goes to my comment on Slack that there may be a simple solution, but _arriving_ at the simple solution may be a bit hard, thus justifying the higher difficulty level.

Reply:
I am happy with either a 2 or 3 rating here.  We may want to make a note of the exercises that are simple at end  but that the simplicity is hard to get to....I don't know if we add more support for them through appends, hints, or more explicit direction in mentor notes -- or all three.  😄 

Another thought with exercises like this is to change the instructions so that they're "stepped out" -- So that they aren't exactly telling the student what to do...but nudging them into a cycle where they move toward that simplicity.

 I am going to do the leveling changes on my machine and re-push a version, since I have to move changed exercises "up" or "down" in ordering.  Are you happy with a 2 here, or would a 3 feel better?

Reply:
I think to keep it a 2 is okay.